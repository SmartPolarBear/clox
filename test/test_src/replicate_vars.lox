fun bad(){
    var a="a";
    var a="b";
    print a;
}

bad();