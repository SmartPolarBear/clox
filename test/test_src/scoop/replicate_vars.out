R"(Error:  at 'a': a already exists in this scoop.)"